### PR TITLE
Move to asynchronous python driver Motor

### PR DESCRIPTION
**Pull request by Themanwhosmellslikesugar**

### CODE REVIEW

File: bin/server.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ def main():
 
     LOGGER.info('RPC server is ready!')
 
-    IOLoop().start(Application(), options.port)
+    IOLoop().start(Application(), 8002)
```
Comment (karpa4o4) [inline bin/server.py]: Unnecessary changes

Comment (eugulixes) [inline bin/server.py:290]: Нужно вернуть на `options.port`.

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 channels==2.4.0
 chardet
 django==2.2.*
+motor==2.3.*
```
Comment (eugulixes) [inline requirements.txt]: Use exact version here (specify the third number, i.e. bugfix).

**Review by eugulixes — APPROVED** [review]
